demo/src/screens/componentScreens/ProgressBarScreen.tsx

```diff
@@ -31,9 +29,7 @@ export default class ProgressBarScreen extends Component {
   }
 
   get customElement() {
-    return (
-      <View style={{backgroundColor: Colors.green30, borderWidth: 0.5, borderRadius: 4}}/>
-    );
+    return <View style={{backgroundColor: Colors.green30, borderWidth: 0.5, borderRadius: 4}}/>;
```

Comment:
Let's keep the PRs as small as possible, changes that are irrelevant to the specific change should be in a different PR or at least a different commit.